about.html

```diff
@@ -13,61 +13,87 @@
 
 <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
+     <header>
+        <div class="topsec">
+          <div class="topimg">
+   <img class="image1" src="img/lambda-black.png">
+          </div>
+        <nav>
+          <a href="index.html">Home</a>
+          <a href="about.html">About</a>
+          <a href="#">Products</a>
+          <a href="#">Blog</a>
+          <a href="#">Contact</a>
+        </nav>
+        </div>
+        <div class="bottomsec">
+        <img class="image2" src="img/jumbo-about.png">
+        </div>
+      </header>
+          <div class= "aboutsec">
+          <h2>About</h2>
+          
+    <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+          </div>
+<section class="mainbody">
+  <div class="content">
+    <img src="img/about-plan.png">
     
```

Comment:
The sections below the header are good places to practice the use of semantic HTML to describe the page better.